### ISSUE TITLE
Draw line if a frame is cropped by the renderer

### ISSUE THREAD

**Issue by NicolasRannou** [FNNDSC/ami#86]
Lik in osirix.

Just need to pass the canvas dimensions as uniform and use gl_FragColor.
```
// Screen size
uniform float screenWidth;
uniform float screenHeight;

...

// Converting (x,y,z) to range [0,1]
float x = gl_FragCoord.x/screenWidth;
float y = gl_FragCoord.y/screenHeight;
float z = gl_FragCoord.z; // Already in range [0,1]
```

![screen shot 2017-01-20 at 11 30 07 am](https://cloud.githubusercontent.com/assets/214063/22146137/f088f548-df03-11e6-8cfb-fd838aa19419.png)

**Comment by NicolasRannou** [comment]
WIP:
![screen shot 2017-01-17 at 6 35 53 pm](https://cloud.githubusercontent.com/assets/214063/22287030/9b50980a-e2f1-11e6-9b61-64350896162b.png)

**Comment by NicolasRannou** [comment]
Closed by ami 0.0.15 - see viewers_upload and xtk_lesson13
